packages/schemas/src/foundations/jsonb-types.ts

```diff
@@ -79,20 +79,12 @@ export const adminConsoleConfigGuard = z.object({
 export type AdminConsoleConfig = z.infer<typeof adminConsoleConfigGuard>;
 
 /**
- * SignIn Experience
+ * SignIn Experiences
  */
 
-export const companyInfoGuard = z.object({
-  name: z.string(),
-  logo: z.string(),
-});
-
-export type CompanyInfo = z.infer<typeof companyInfoGuard>;
-
 export enum BrandingStyle {
-  CompanyLogo_CompanyName_AppName = 'CompanyLogo_CompanyName_AppName',
-  CompanyLogo_AppLogo_CompanyName_AppName = 'CompanyLogo_AppLogo_CompanyName_AppName',
-  AppLogo_CompanyName_AppName = 'AppLogo_CompanyName_AppName',
+  Logo_Slogan = 'Logo_Slogan',
```

Comment:
LogoSlogan?

Reply:
Need `_` to split different terminologies, and make it clear. e.g. `'AppLogo_CompanyName_AppName'`.

Reply:
Conclusion: Prefer Option B

A.
```ts
LogoSlogan = 'Logo_Slogan',
AppLogoCompanyNameAppName = 'AppLogo_CompanyName_AppName',
```
B.
```ts
Logo_Slogan = 'Logo_Slogan',
AppLogo_CompanyName_AppName = 'AppLogo_CompanyName_AppName',
```

See [discussion](https://silverhand-io.slack.com/archives/C02A8G4HVAM/p1646984386183799) on Slack.